src/browser_app_skeleton/public/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
```

Comment:
Can you add a comment just above this pointing to https://www.robotstxt.org/robotstxt.html ? Just a quick little reference saying "For info on updating this" or whatever.

Reply:
Done :+1: